package.json

```diff
@@ -236,6 +237,7 @@
     "react-router": "5.2.0",
     "react-router-dom": "5.2.0",
     "react-svg-inline": "2.1.1",
+    "react-use": "^17.2.1",
```

Comment:
@daniloprates please set fixed version

Reply:
This one isn't used anymore (was used in the confetti fx)